### ISSUE TITLE
Initial Setup failing

### ISSUE THREAD

**Issue by spadiyar** [jainvipin/tutorial#3]
I am not able to go through the container networking tutorial as it gets stuck at the stage below and on ctrl-c echoes error below:

PANMH273G3QCHQ:vagrant-setup spadiyar$ 
PANMH273G3QCHQ:vagrant-setup spadiyar$ 
PANMH273G3QCHQ:vagrant-setup spadiyar$ ls
Vagrantfile resolv.conf
PANMH273G3QCHQ:vagrant-setup spadiyar$ !vag
vagrant up
Bringing machine 'tutorial-node1' up with 'virtualbox' provider...
Bringing machine 'tutorial-node2' up with 'virtualbox' provider...
==> tutorial-node1: Box 'contiv/ubuntu1504-netplugin' could not be found. Attempting to find and install...
    tutorial-node1: Box Provider: virtualbox
    tutorial-node1: Box Version: 0.3.1

.......
.......
stuck here for several minutes.

I hit <ctrl-c>

The box 'contiv/ubuntu1504-netplugin' could not be found or
could not be accessed in the remote catalog. If this is a private
box on HashiCorp's Atlas, please verify you're logged in via
`vagrant login`. Also, please double-check the name. The expanded
URL and error message are shown below:

URL: ["https://atlas.hashicorp.com/contiv/ubuntu1504-netplugin"]
Error: Operation timed out after 0 milliseconds with 0 out of 0 bytes received
PANMH273G3QCHQ:vagrant-setup spadiyar$ 
PANMH273G3QCHQ:vagrant-setup spadiyar$

**Comment by jainvipin** [comment]
@spadiyar - thanks for giving it a try; couple of things to look at:

I tried it just now it worked (is it possible that atlas/hashicorp link was down then when you tried it)?

I am also curious if you are trying this out on a Windows machine? Because it works for me on a MAC.

What is the version of Vagrant and Virtualbox that you are running?

**Comment by spadiyar** [comment]
I am trying it on mac and still hit the same error - 

Virtual box version is 5.1.4
Vagrant is 1.8.5

Is there a way to just run contiv on a ubuntu server without the whole vagrant and virtualbox setup on mac.

PANMH273G3QCHQ:vagrant-setup spadiyar$ vagrant up
Bringing machine 'tutorial-node1' up with 'virtualbox' provider...
Bringing machine 'tutorial-node2' up with 'virtualbox' provider...
==> tutorial-node1: Box 'contiv/ubuntu1504-netplugin' could not be found. Attempting to find and install...
    tutorial-node1: Box Provider: virtualbox
    tutorial-node1: Box Version: 0.3.1
^C==> tutorial-node1: Waiting for cleanup before exiting...
The box 'contiv/ubuntu1504-netplugin' could not be found or
could not be accessed in the remote catalog. If this is a private
box on HashiCorp's Atlas, please verify you're logged in via
`vagrant login`. Also, please double-check the name. The expanded
URL and error message are shown below:

URL: ["https://atlas.hashicorp.com/contiv/ubuntu1504-netplugin"]
Error: 
PANMH273G3QCHQ:vagrant-setup spadiyar$

**Comment by jainvipin** [comment]
@spadiyar - hmm... it should have worked without a hitch. Is there any chance you are running this behind a proxy (http/https proxy)? If you you may have to configure those things too.

On the other hand it is possible to run contiv on ubuntu server (or a set of them as a cluster) but I am not sure if that'd be any different from this. I can point you to those instructions if needed.

**Comment by spadiyar** [comment]
Yeah its possible, let me dig into that, also will check privileges needed on the mac in case some things in the script are failing due to that. Please send me the instructions for ubuntu, i already have docker installed on the ubuntu servers so can skip those steps.

**Comment by jainvipin** [comment]
@spadiyar - if you are  behind proxy, you can easily check if `curl https://www,google.com` works for you; if not, you'd still need https_proxy or http_proxy settings, even if you have pre-built VMs. Vagrant/Vboxes are much simpler to try and make work, regardless the instructions to install and run on a vanilla server (installed docker binary engine is not a big issue) is:
http://contiv.github.io/documents/gettingStarted/networking/install-swarm.html

**Comment by spadiyar** [comment]
ok i ran the steps in the vagrant file manually and it appears git clone is failing. Do i need any specific permissions to clone your tutorial ?

PANMH273G3QCHQ:opt spadiyar$ git clone https://github.com/jainvipin/tutorial
fatal: could not create work tree dir 'tutorial': Permission denied
PANMH273G3QCHQ:opt spadiyar$

**Comment by jainvipin** [comment]
absolutely not @spadiyar - it is public repo that anyone shoudl be able to clone. It seems like the permissions to create a new directory in the current path doesn't seem to be there... if you join the contiv slack channel at https://contiv-slack.herokuapp.com/ we can close this quicker.

**Comment by spadiyar** [comment]
i am in the slack channel now
